<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReNameWin</class>
 <widget class="QWidget" name="ReNameWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>256</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>470</width>
    <height>260</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ReNameBatchWin</string>
  </property>
  <property name="windowIcon">
   <iconset resource="RealCompare.qrc">
    <normaloff>:/Resources/edit/global/notebook.png</normaloff>:/Resources/edit/global/notebook.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Select Dir</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditDir"/>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonBrowse">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="optionsTab">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="addDelName">
      <attribute name="title">
       <string>AddDelNameString</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QRadioButton" name="radioButtonAddPrefix">
         <property name="text">
          <string>Add Prefix</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditAddPrefix">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QRadioButton" name="radioButtonDelPrefix">
         <property name="text">
          <string>Del Prefix</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLineEdit" name="lineEditDelPrefix">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="radioButtonAddSuffix">
         <property name="text">
          <string>Add Suffix</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditAddSuffix">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QRadioButton" name="radioButtonDelSuffix">
         <property name="text">
          <string>Del Suffix</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLineEdit" name="lineEditDelSuffix">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="radioButtonLower">
         <property name="text">
          <string>Lower FileName</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QRadioButton" name="radioButtonUpper">
         <property name="text">
          <string>Upper FileName</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="changeExt">
      <attribute name="title">
       <string>ChangeExt</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Modify Ext To</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditDestExt"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Deal Ext Type</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxExt">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>All File Ext Type</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="definedFilterExt">
           <property name="text">
            <string>defined</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxDealChildDir">
     <property name="text">
      <string>Deal Child Dir</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="startBt">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeBt">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="RealCompare.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeBt</sender>
   <signal>clicked()</signal>
   <receiver>ReNameWin</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>456</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonBrowse</sender>
   <signal>clicked()</signal>
   <receiver>ReNameWin</receiver>
   <slot>slot_selectDir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>420</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startBt</sender>
   <signal>clicked()</signal>
   <receiver>ReNameWin</receiver>
   <slot>slot_startRename()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>definedFilterExt</sender>
   <signal>clicked()</signal>
   <receiver>ReNameWin</receiver>
   <slot>slot_userDefineExt()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>457</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_selectDir()</slot>
  <slot>slot_startRename()</slot>
  <slot>slot_userDefineExt()</slot>
 </slots>
</ui>
